.circleci/config.yml

```diff
@@ -1,12 +1,31 @@
-version: 2
+version: 2.1
+
 jobs:
-  build:
+  all:
     docker:
-      - image: circleci/node:8.12.0
+      - image: circleci/node:carbon
     steps:
       - checkout
       - run: npm install
       - run: npm run lint
       - run: npm test
       - run: npm run build
       - run: npm run size
+      - run: npm run integration
+
+workflows:
+  version: 2
+  commit:
+    jobs:
+      - all
+  schedule:
+    jobs:
+      - all
+    triggers:
+      - schedule:
+          cron: "40 * * * *"
```

Comment:
Can you please add a comment here what this configuration means and how often the job will be run?